test/testdrive/statement-logging.td

```diff
@@ -91,7 +91,7 @@ contains: column "f_nonexistent" does not exist
        test_begin AS (SELECT began_at FROM all_stmts WHERE sql = 'SELECT ''beginning real test!''' ORDER BY began_at DESC LIMIT 1)
   SELECT all_stmts.sample_rate, all_stmts.params, all_stmts.finished_status, all_stmts.error_message, all_stmts.rows_returned, all_stmts.execution_strategy, all_stmts.name LIKE 's%', all_stmts.sql
   FROM all_stmts, test_begin WHERE all_stmts.began_at >= test_begin.began_at AND all_stmts.sql NOT LIKE '%sduiahsdfuoiahsdf%' --ignore this statement, in case we have to re-run it
-  ORDER BY all_stmts.sql, all_statements.rows_returned
+  ORDER BY all_stmts.sql, all_stmts.rows_returned
```

Comment:
I fixed this test as part of [6fe7111](https://github.com/MaterializeInc/materialize/commit/6fe7111f29a876f5121c50d4cc1ab1cef7f4432b), so the easiest thing for you to do to avoid merge conflicts would be to revert your changes to this file and then rebase on origin/main.